internal/transport/http2_client.go

```diff
@@ -447,7 +450,7 @@ func newHTTP2Client(connectCtx, ctx context.Context, addr resolver.Address, opts
 		err := t.loopy.run()
 		if err != nil {
 			if logger.V(logLevel) {
-				logger.Errorf("transport: loopyWriter.run returning. Err: %v", err)
+				logger.Infof("transport: loopyWriter exited. Closing connection. Err: %v", err)
```

Comment:
I think we should avoid package names in logs messages. We do have a plan to use a prefix logger for every transport instance. So, having this package prefix would become meaningless at that point.

Package prefixes do make sense for error strings.

@dfawley : Thoughts on this?

Reply:
Let's leave it all `transport: ` prefixed since that's our existing convention, and update it to use a prefix logger in a follow-on change.